devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -266,13 +266,18 @@ public void execute() throws MojoFailureException, MojoExecutionException {
             if (plugin == null) {
                 throw new MojoExecutionException("Failed to locate " + key + " among the project plugins");
             }
+            Xpp3Dom configuration = (Xpp3Dom) plugin.getConfiguration();
+            if (configuration == null) {
+                // TODO: Set sources and target to supported version
```

Comment:
Should we keep this TODO or do it right away?

I think I would go with 8 for now and upgrade to 11 as soon as we drop 8.

WDYT?